qa/workunits/rados/test.sh

```diff
@@ -29,6 +29,7 @@ for f in \
     api_service api_service_pp \
     api_c_write_operations \
     api_c_read_operations \
+    api_cls_remote_reads \
```

Comment:
this work unit is picked up by more than one task yaml file. `qa/suites/rados/basic/tasks/rados_api_tests.yaml` is but one of them. the failed test was
```
rados/verify/{centos_latest ceph clusters/{fixed-2 openstack} d-thrash/none mon_election/classic msgr-failures/few msgr/async objectstore/bluestore-comp-zstd rados tasks/rados_api_tests validater/lockdep}
```

where `rados/verify/tasks/rados_api_tests` is used as one of the workloads. i'd suggest
```sh
git grep -w rados/test.sh | grep suites
```
for finding all the yml file which uses the changed work unit. but there are couple other ways to address this issue

- probably to update the settings right in `qa/workunits/rados/test.sh`.
- add a new .sh file exercising `api_cls_remote_reads` in `qa/workunits/cls/. because `rados_cls_all.yaml` already takes care of this by enabling all osd classes.